webpack/scenes/Hosts/ChangeContentSource/index.js

```diff
@@ -74,6 +80,13 @@ const ChangeContentSourcePage = () => {
     }
   };
 
+  const breadcrumbItems = () => {
```

Comment:
If it's a single host the breadcrumb should be

```
Hosts > hostname > Change host content source
```

with "Hosts" and "hostname" as links.
If it's multiple hosts the breadcrumb can remain

```
Hosts > Change host content source
```
with "Hosts" as the link.

@MariSvirik can check me, but I think this is how breadcrumbs should be.